docs/docs/howto/clusterconnectivity/aiven-kafka-cluster-ssl-protocol.rst

```diff
@@ -7,7 +7,7 @@ This section provides you with information on how to connect Aiven for Apache Ka
 Prerequisite
 ------------
 
-* Set up the connection between the Klaw APIs (Core API and Cluster API), see :doc:`klaw-core-with-clusterapi`. This involves configuring the ``klaw.clusterapi.url`` setting in the Klaw UI and testing the connectivity to ensure the two APIs can communicate.
+* Set up the connection between the Klaw APIs (Core API and Cluster API) to use secure SSL, see :doc:`klaw-core-with-clusterapi`. This involves configuring the ``klaw.clusterapi.url`` setting in the Klaw UI and testing the connectivity to ensure the two APIs can communicate over https.
```

Comment:
As kafka ssl connectivity does not use application's security context, this should not be a pre.